DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Suggests:
     knitr,
     labelled,
     metacore,
+    pharmaverseadam,
```

Comment:
i don't think this is needed

Reply:
I think the data from the package is used in the tests now - so it is needed

Reply:
yep, sorry i missed that!